x-pack/legacy/plugins/infra/public/containers/logs/log_analysis/log_analysis_results_url_state.tsx

```diff
@@ -40,12 +40,9 @@ export const useLogAnalysisResultsUrlState = () => {
       pipe(urlTimeRangeRT.decode(value), fold(constant(undefined), identity)),
     encodeUrlState: urlTimeRangeRT.encode,
     urlStateKey: TIME_RANGE_URL_STATE_KEY,
+    writeDefaultState: true,
```

Comment:
:information_source: The initialization is now handled by the `useUrlState` hook itself.